ubyssey/helpers.py

```diff
@@ -164,3 +170,65 @@ def get_random_articles(n, section, exclude=None):
                 results.append(articles[index])
 
         return results
+
+class EventsHelper(object):
+
+    @staticmethod
+    def get_calendar_events(category=None, months=None, start=None, end=None):
+        events = Event.objects \
+            .filter(is_submission=False) \
+            .filter(is_published=True) \
+            .order_by('start_time')
+
+        today = date.today()
+        # filter start
+        if start is not None:
+            events = events.filter(start_time__gt=start)
+        else:
+            events = events.filter(start_time__gt=today)
+
+        # filter end
+        if end is not None:
+            events = events.filter(end_time__lte=end)
+        else:
+            until_month = today.month + (months if months is not None else 12)
+            until_year = today.year
+            while until_month > 12:
+                until_month -= 12
+                until_year += 1
+            dt_until = today.replace(year=until_year, month=until_month)
+            events = events.filter(end_time__lte=dt_until)
+
+        if category is not None and category != 'all':
+            events = events.filter(category__exact=category)
+
+        HARD_MAX = 100
+        events = events[:HARD_MAX]
+
+        return events
+
+    @staticmethod
+    def organize_events_by_date(events):
+        events_by_date = OrderedDict()
+
+        for event in events:
+            start = event.start_time # Zulu
+            start = start.astimezone(timezone('America/Vancouver'))
```

Comment:
Would it be possible to use the timezone defined in settings.py? I'm just worried about hard-coding this.